File: june14/july7ev_Rec5/pkpc.cpp
#include<iostream>
#include<string>

using namespace std;

string codes[10]={".","abc","def","ghi","jkln","no","pqrs","tu","vwxy","z"};

void pkpc(string q,string a)
{
    if(q.length()==0)
    {
        cout << a << endl;
        return;
    }
    char ch=q[0];
    string roq=q.substr(1);
    string code=codes[ch-'0'];
    for(int i=0;i<code.length();i++)
        pkpc(roq,a+code[i]);
}
int main(int argc,char** argv)
{
    pkpc("835","");
}

File: feb2/mar2_Rec4/pkpc.java
package mar2;

public class pkpc {

	static String[] codes={".;","abc","def","ghi","jkl","mno","pqrs","tuv","wx","yz"};
	
	public static void pkpc(String ques,String asf){
		if(ques.length()==0){
			System.out.print(asf+" ");
			return;
		}
		char ch=ques.charAt(0);
		int l=ch-'0';
		String roq=ques.substring(1);
		for(int i=0;i<codes[l].length();i++){
			pkpc(roq,asf+codes[l].charAt(i));
		}
		
	}
	
	public static void main(String[] args) {
		pkpc("682","");

	}

}
